yb-voyager/cmd/common.go

```diff
@@ -204,7 +204,7 @@ func CreateMigrationProjectIfNotExists(dbType string, exportDir string) {
 	// TODO: add a check/prompt if any directories apart from required ones are present in export-dir
 	var projectSubdirs = []string{
 		"schema", "data", "reports",
-		"metainfo", "metainfo/data", "metainfo/schema", "metainfo/flags",
+		"metainfo", "metainfo/data", "metainfo/schema", "metainfo/flags", "metainfo/conf",
```

Comment:
Have you confirmed that the directory gets cleaned when you use `--start-clean` flag?

Reply:
Made changes to remove that file in export data start-clean flow 👍  // cc @sanyamsinghal